SPECS-EXTENDED/nss-mdns/nss-mdns.spec

```diff
@@ -0,0 +1,229 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+
+Name: nss-mdns
+Version: 0.15.1
+Release: 11%{?dist}
+Summary: glibc plugin for .local name resolution
+
+License: LGPLv2+
+URL: https://github.com/lathiat/nss-mdns
+Source0: https://github.com/lathiat/nss-mdns/releases/download/v%{version}/nss-mdns-%{version}.tar.gz
+
+# https://github.com/lathiat/nss-mdns/pull/84
+Patch1:  nss-mdns-local-heuristic.patch
+Patch2:  nss-mdns-local-heuristic-unit.patch
+
+BuildRequires: make
+BuildRequires: gcc
+BuildRequires: pkgconfig(check)
+Requires: avahi
+Requires: authselect
+
+%description
+nss-mdns is a plugin for the GNU Name Service Switch (NSS) functionality of
+the GNU C Library (glibc) providing host name resolution via Multicast DNS
+(aka Zeroconf, aka Apple Rendezvous, aka Apple Bonjour), effectively allowing
+name resolution by common Unix/Linux programs in the ad-hoc mDNS domain .local.
+
+nss-mdns provides client functionality only, which means that you have to
+run a mDNS responder daemon separately from nss-mdns if you want to register
+the local host name via mDNS (e.g. Avahi).
+
+
+%prep
+%autosetup -p1
+
+%build
+%configure --libdir=/%{_lib}
+%make_build
+
+%check
+make check || (cat ./test-suite.log; false)
+
+%install
+rm -rf $RPM_BUILD_ROOT
+%make_install
+
+
+%post
+%{?ldconfig}
+
+%posttrans
+authselect enable-feature with-mdns4 &> /dev/null || :
```

Comment:
I have triggered a test build:
[PR-7407](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=494569&view=results)

A note about the the prior test builds, they do `build` the package, but fail on `authselect` as it is a rutime dependency.

Have we verified the working of this package post a build ?

Reply:
Package was installed with no issues the only thing was to add `extended repo` from where `authselect` would come.
<img width="1113" alt="install" src="https://github.com/microsoft/CBL-Mariner/assets/14061455/91501c58-416a-460c-bce4-c8e2bd476933">
All the modules after installation exists in the `lib` dir [reference](https://github.com/avahi/nss-mdns?tab=readme-ov-file#libraries)
<img width="538" alt="libs" src="https://github.com/microsoft/CBL-Mariner/assets/14061455/0d7a36dd-2ebb-44db-bca1-8b5f6738d1b4">
Plugin activation and proof of working [reference](https://github.com/avahi/nss-mdns?tab=readme-ov-file#activation)
<img width="547" alt="resolving" src="https://github.com/microsoft/CBL-Mariner/assets/14061455/53d0c40e-a2e2-4324-acc6-96d438a6117f">